### PR TITLE
Done Pundit's Card

### PR DESCRIPTION
**Pull request by gussiciliano**
### Sumary
- Add permissions with pundit

### Trello Card
https://trello.com/c/2Khe2mQM/12-instalar-y-configurar-pundit

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -17,4 +19,46 @@ def set_locale
     I18n.locale = current_user.try(:locale) || I18n.default_locale
   end
 
+  def authentication_manager
```
Comment (Dinuuu) [inline app/controllers/application_controller.rb]: all this code shouldn't be in the applicationController, didn't you move it to an AuthorizatedController or something like that in the active admin PR ?

Comment (gussiciliano) [inline app/controllers/application_controller.rb]: Oks, yes, probably a do this with some rebase jaja

File: spec/policies/rent_policy_spec.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
```
Comment (Dinuuu) [inline spec/policies/rent_policy_spec.rb]: delete this file if you are not going to use it

Comment (gussiciliano) [inline spec/policies/rent_policy_spec.rb]: Oks

**Pull request merged**